src/nvim/file_search.c

```diff
@@ -1310,15 +1310,17 @@ static int ff_path_in_stoplist(char_u *path, int path_len, char_u **stopdirs_v)
 char_u *
 find_file_in_path (
     char_u *ptr,               /* file name */
-    int len,                        /* length of file name */
+    size_t len,                /* length of file name */
     int options,
     int first,                      /* use count'th matching file name */
     char_u *rel_fname         /* file name searching relative to */
 )
 {
   return find_file_in_path_option(ptr, len, options, first,
-      *curbuf->b_p_path == NUL ? p_path : curbuf->b_p_path,
-      FINDFILE_BOTH, rel_fname, curbuf->b_p_sua);
+                                  *curbuf->b_p_path == NUL
+                                  ? p_path
+                                  : curbuf->b_p_path,
+                                  FINDFILE_BOTH, rel_fname, curbuf->b_p_sua);
```

Comment:
It's different because `?` and `:` are subexpressions of the ternary condition. I believe we *do* indent nested `&&` and `+`, e.g.:

```c
if (foo
  && (bar
    && baz)) {
    ...
```

instead of:

```c
if (foo
  && (bar
  && baz)) {
    ...
```

If you don't agree, don't worry about it.